### DISCUSSION TITLE
Can Markdown footnotes start with a selected number?

### DISCUSSION THREAD

**Discussion by glow12121** [datenstrom/community#812] [Ask a question] [answered]
Hi,

On my website, I have a fragment from a book featuring multiple footnotes. They do not start with `[1]` but with a different number instead. But in the end, in the processed html form, the footnotes still start with 1 - in the text and in the end explanations.

Can I make them start with a number other than 1?

**Comment by GiovanniSalmeri** [comment, marked as answer]
To the best of my knowledge, unfortunately neither PHP Markdown nor Parsedown (the two parsers which can be used in Yellow) have such a capability: no matter which labels are used in footnotes, footnotes are numbered from 1. Modifying the parsers is possible, but doesn't seem to me a very good idea.

The simplest solution seems to me: copying the HTML output of the fragment, modifying the numbers of the references (this could be done automagically with the text editor Vim, for example), adding a `start` attribut to the `<ol>` tag, and inserting this fragment in the page as HTML.

**Reply by glow12121** [reply]
Thank you!

I guess doing this every time I render the website is too complicated, so I will keep the numbering as it is.

**Reply by GiovanniSalmeri** [reply]
Just for the sake of clarity: you can insert HTML in a markdown page: not very elegant, but it works (and it's in the spirit of Markdown). So this manipulation should be done only once for each book fragment which has this problem.

**Reply by markseuffert** [reply]
In addition to Giovanni's excellent advice to use [HTML](https://www.w3schools.com/html/), you can also make a new extension with the [API for developers](https://datenstrom.se/yellow/help/api-for-developers). As a developer you can extend Markdown with shortcuts. For example something like `[footnote 4 "Be kind, for everyone you meet is fighting a hard battle. - Sokrates"]` and `[footnote 5 "Wonder is the beginning of wisdom. - Sokrates”]`. This would give you the opportunity to insert footnotes with a selected number. Maybe with an option to insert footnotes with letters? Maybe with the option to show footnotes at a specific position on the page? The API has you covered. 😁

**Reply by glow12121** [reply]
@GiovanniSalmeri , I got it, thank you -- will try this when I have a free minute.

@markseuffert , my expertise in programming matters is almost zero.
